b_cfn_elasticsearch_cloner/cloner_source/index.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import os
+from typing import Any, Dict
+
+import boto3
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+from elasticsearch.helpers import bulk
+from requests_aws4auth import AWS4Auth
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+try:
+    from dynamodb_decode import DynamodbDecoder
+except ImportError as ex:
+    logger.exception("Failed import.")
+    from b_cfn_elasticsearch_cloner.cloner_source.dynamodb_decode import DynamodbDecoder
+
+
+es_index = os.environ["ES_INDEX_NAME"]
+es_endpoint = os.environ["ES_DOMAIN_ENDPOINT"]
+primary_key_field = os.environ["PRIMARY_KEY_FIELD"]
+region = os.environ["AWS_REGION"]
+
+
+credentials = boto3.Session().get_credentials()
+awsauth = AWS4Auth(
+    credentials.access_key,
+    credentials.secret_key,
+    region,
+    "es",
+    session_token=credentials.token,
+)
+
+es = Elasticsearch(
+    hosts=[{"host": es_endpoint, "port": 443}],
+    http_auth=awsauth,
+    use_ssl=True,
+    verify_certs=True,
+    connection_class=RequestsHttpConnection,
+)
+
+def handler(event: Dict[Any, Any], context: Any) -> Dict[str, Any]:
```

Comment:
Since this is an entry point of the custom resource, please add a docstring telling exactly what this function (handler does).